src/Analyzers/Core/CodeFixes/NamingStyle/NamingStyleCodeFixProvider.cs

```diff
@@ -108,9 +108,12 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
         private class FixNameCodeAction : CodeAction
         {
+#pragma warning disable IDE0052 // Remove unread private members - used in non-CodeStyle project context
             private readonly Solution _startingSolution;
             private readonly ISymbol _symbol;
             private readonly string _newName;
+#pragma warning restore IDE0052 // Remove unread private members
```

Comment:
should we instead have `#if !CODE_STYLE` then?

Reply:
Sure, I can take that approach.